pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -340,9 +336,9 @@ def make_package(args):
         with open(args.intent_filters) as fd:
             args.intent_filters = fd.read()
 
-    if get_bootstrap_name() == "sdl2":
-        args.add_activity = args.add_activity or []
-        args.activity_launch_mode = args.activity_launch_mode or ''
+    # if get_bootstrap_name() == "sdl2":
+    args.add_activity = args.add_activity or []
+    args.activity_launch_mode = args.activity_launch_mode or ''
```

Comment:
Minor: I know it was already that way before, but people misuse this as ternary operator, this is not the python ternary operator

Reply:
So with Python-style ternary that would be:
```
args.add_activity = args.add_activity if args.add_activity else []
args.activity_launch_mode = args.activity_launch_mode if args.activity_launch_mode or ''
```